common/src/main/java/com/wynntils/models/containers/ContainerModel.java

```diff
@@ -42,9 +42,11 @@ public final class ContainerModel extends Model {
     private static final Pair<Integer, Integer> BANK_PREVIOUS_NEXT_SLOTS = new Pair<>(17, 8);
     private static final Pair<Integer, Integer> GUILD_BANK_PREVIOUS_NEXT_SLOTS = new Pair<>(9, 27);
     private static final Pair<Integer, Integer> TRADE_MARKET_PREVIOUS_NEXT_SLOTS = new Pair<>(17, 26);
+    private static final Pair<Integer, Integer> SCRAP_MENU_PREVIOUS_NEXT_SLOTS = new Pair<>(0, 8);
     private static final CodedString LAST_BANK_PAGE_STRING = CodedString.fromString(">§4>§c>§4>§c>");
     private static final CodedString FIRST_TRADE_MARKET_PAGE_STRING = CodedString.fromString("§bReveal Item Names");
     private static final CodedString TRADE_MARKET_TITLE = CodedString.fromString("Trade Market");
+    private static final CodedString SCRAP_MENU_TITLE = CodedString.fromString("Scrap Rewards");
```

Comment:
I know the other parts are not ported, but can you try to use StyledText in cases where it is not a hassle? (Like here)

Reply:
(Otherwise we are never replacing CodedString)